code-gen/src/main/scala/scalapb/grpcweb/GrpcWebServicePrinter.scala

```diff
@@ -36,7 +36,7 @@ final class GrpcWebServicePrinter(
       case StreamType.Unary =>
         s"${method.deprecatedAnnotation}def ${method.name}" + s"(request: ${method.inputType.scalaType}$contextParam): scala.concurrent.Future[${method.outputType.scalaType}]"
       case StreamType.ServerStreaming =>
-        s"${method.deprecatedAnnotation}def ${method.name}" + s"(request: ${method.inputType.scalaType}$contextParam, responseObserver: ${observer(method.outputType.scalaType)}): Unit"
+        s"${method.deprecatedAnnotation}def ${method.name}" + s"(request: ${method.inputType.scalaType}$contextParam, responseObserver: ${observer(method.outputType.scalaType)}): _root_.scalapb.grpcweb.native.ClientReadableStream"
```

Comment:
To avoid leaking the native object ot users, and for future compatibility with the possibility of client streaming and bidi, can you change this to return `io.grpc.stub.ClientCallStreamObserver`. The only method to implement for this abstract class should be `cancel()`, the rest should be `???`.